Author: <bdefreese@debian2.bddebian.com>
Description: Correctly 0 out the c struct.
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

--- a/msgsnarf.c
+++ b/msgsnarf.c
@@ -584,6 +584,7 @@
 		if (i == 0) {
 			if ((c = malloc(sizeof(*c))) == NULL)
 				nids_params.no_mem("sniff_msgs");
+			memset(c, 0, sizeof(*c));
 			c->ip = ts->addr.saddr;
 			c->nick = strdup("unknown");
 			SLIST_INSERT_HEAD(&client_list, c, next);
